lib/govuk_index/presenters/common_fields_presenter.rb

```diff
@@ -81,6 +81,18 @@ def section_id
       @_section_id ||= payload.dig("details", "section_id") if format == "hmrc_manual_section"
     end
 
+    def is_historic
+      is_political && payload.dig("expanded_links", "government", 0, "details", "current") == false
+    end
+
+    def is_political
+      payload.dig("details", "political") || false
+    end
+
+    def government_name
+      payload.dig("expanded_links", "government", 0, "title")
+    end
```

Comment:
I wonder if you want a helper method of government that can return: `payload.dig("expanded_links", "government").first`

and then government_name could be `government && government.dig("details", "title")` and is historic could be `is_political && government && government["current"]`